src/components/Note.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useContext, useState } from 'react';
+import styled from 'styled-components';
+import { mainStateDispatchContext } from './Main';
+import { GithubPicker } from 'react-color';
+import { Icon } from 'semantic-ui-react';
+import brightnessByColor from './brightnessByColor';
+
+const icons = {
+  delete: <Icon name="delete" />,
+  cancel: <Icon name="ban" />,
+  save: <Icon name="save" />,
+};
+
+const NoteWrapper = styled.div`
+  border:1px solid;
+  border-bottom-right-radius: 16px;
+  display: flex;
+  flex-direction: column;
+  button {
+      background: none;
+      border: none;
+      outline: none;
+  }
+`;
+
+const NoteToolbar = styled.div`
+  background-color: rgba(256,256,256,0.7);
+  padding: 5px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+`;
+
+const NoteBody = styled.div`
+  flex-grow: 1;
+  textarea {
+    width: 97%;
```

Comment:
97?? how did u come up with a prime number? :)

Dont use these numbers. Its not a professional practice

Reply:
Done